model_compression_toolkit/core/tpc_models/get_target_platform_capabilities.py

```diff
@@ -68,16 +39,20 @@ def get_target_platform_capabilities(fw_name: str,
     Args:
         fw_name: Framework name of the TargetPlatformCapabilities.
         target_platform_name: Target platform model name the model will use for inference.
-
+        target_platform_version: Target platform capabilities version.
     Returns:
         A TargetPlatformCapabilities object that models the hardware and attaches
         a framework information to it.
     """
-    assert fw_name in tpc_dict, f'Framework {fw_name} is not supported'
-    supported_models_by_fw = tpc_dict.get(fw_name)
-    assert supported_models_by_fw is not None, f'Supported models for {fw_name} is empty. Make sure the relevant ' \
-                                               f'packages are installed when using MCT for optimizing a {fw_name} model. ' \
-                                               f'For Tensorflow, please install tensorflow and tensorflow-model-optimization. ' \
-                                               f'For PyTorch, please install torch.'
-    assert target_platform_name in supported_models_by_fw, f'Target platform model named {target_platform_name} is not supported for framework {fw_name}'
-    return tpc_dict.get(fw_name).get(target_platform_name)
+    assert target_platform_name in tpc_dict, f'Target platform {target_platform_name} does not defined!'
+    fw_tpc = tpc_dict.get(target_platform_name)
+    assert fw_name in fw_tpc, f'Framework {fw_name} is not supported in {target_platform_name}. Please make sure the relevant ' \
+                              f'packages are installed when using MCT for optimizing a {fw_name} model. ' \
+                              f'For Tensorflow, please install tensorflow and tensorflow-model-optimization. ' \
+                              f'For PyTorch, please install torch.'
+    tpc_versions = fw_tpc.get(fw_name)
+    if target_platform_version is None:
+        target_platform_version = 'latest'
+    else:
```

Comment:
If we use this scheme, please use 'latest' as a constant (here and in the creation of the dictionaries).

Reply:
done